src/components/Navbar.js

```diff
@@ -1,12 +1,18 @@
-import { useLocation } from "react-router";
-import { LOGIN_PAGE, REGISTER_PAGE } from "../constants";
+import { useLocation, useNavigate } from "react-router";
+import { LOGIN_PAGE, REGISTER_PAGE, ADD_MOVIE_PAGE, HOME_PAGE } from "../constants";
 import useAuth from "./hooks/useAuth";
 import NavLink from "./NavLink";
 
 function Navbar() {
   const location = useLocation();
+  const navigate = useNavigate();
   const { logout } = useAuth();
 
+  function reloadHomePage(){
```

Comment:
jel ovo mora?

Reply:
Ovo mi je trebalo zbog nekog bug-a.. Pokusacu sad da ispravim to

Reply:
I meni je fuj